backend/src/main/kotlin/blue/mild/covid/vaxx/dao/model/User.kt

```diff
@@ -3,6 +3,8 @@ package blue.mild.covid.vaxx.dao.model
 import org.jetbrains.exposed.sql.Table
 import org.jetbrains.exposed.sql.`java-time`.timestamp
 
+// lengths in the database
+@Suppress("MagicNumber")
 object User : Table("users") {
     val id = varchar("id", 36)
```

Comment:
Udelal bych konsktantu DB_ID_LENGTH = 16, pac je to pouzite pomalu ve vsech tabulkach